test/test_matmul_default.cpp

```diff
@@ -0,0 +1,115 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#include <csrc/exceptions.h>
+#include <gtest/gtest.h>
+
+#include <fusion.h>
+#include <mma_type.h>
+#include <ops/all_ops.h>
+#include <scheduler/all_schedulers.h>
+#include <scheduler/mma_utils.h>
+#include <test/utils.h>
+#include <test/validator.h>
+
+namespace nvfuser {
+
+class MatmulDefaultTest : public NVFuserTest {};
+
+TEST(MatmulDefaultTest, SingleMmaOp) {
+  auto fusion = std::make_unique<Fusion>();
+  FusionGuard fg(fusion.get());
+  EnableOptionsGuard::getCurOptions().set(EnableOption::MatmulExprEval);
```

Comment:
You missed saving and restoring the previous state. See https://github.com/NVIDIA/Fuser/blob/72198a74661bc7d0c4e6185a5b302f9128ff978b/test/test_alias.cpp#L1113-L1114 and https://en.cppreference.com/w/cpp/language/raii.